hammerspoon/microphone.lua

```diff
@@ -0,0 +1,60 @@
+local message = require('status-message')
+
+local messageMuting = message.new('muted 🎤')
+local messageHot = message.new('hot 🎤')
+local last_mods = {}
+local recently_clicked = false
+local second_click = false
+
+display_status = function ()
+  -- Check if the active mic is muted
+  if hs.audiodevice.current(true).device:muted() then
+    messageMuting:notify()
+  else
+    messageHot:notify()
+  end
+end
+display_status()
+
+toggle = function (device)
+  if device:muted() then
+    device:setMuted(false)
+  else
+    device:setMuted(true)
+  end
+end
+
+control_key_handler = function()
+  recently_clicked = false
+end
+
+control_key_timer = hs.timer.delayed.new(0.3, control_key_handler)
```

Comment:
I suspect that the name `control_key_timer` is copy-and-paste from [`control_escape.lua`](https://github.com/jasoncodes/dotfiles/blob/ac9f3ac/hammerspoon/control_escape.lua). Since we're dealing with the option key here, I think `optionKeyHandler` would be a more intention-revealing name.